pallets/automation-time/src/mock.rs

```diff
@@ -351,7 +351,11 @@ impl Convert<CurrencyId, Option<MultiLocation>> for MockTokenIdConvert {
 pub struct MockEnsureProxy;
 impl EnsureProxy<AccountId> for MockEnsureProxy {
 	fn ensure_ok(_delegator: AccountId, _delegatee: AccountId) -> Result<(), &'static str> {
-		Ok(())
+		if _delegator == ALICE.into() && _delegatee == BOB.into() {
```

Comment:
If this ALICE is not used anywhere else we should call it `OWNER_ACCOUNT` and BOB for `PROXY_ACCOUNT`, where OWNER_ACCOUNT is the delegator and PROXY_ACCOUNT is a delegatee.